<!-- Start:AutoDoc:: Modify `src/readme/rules.ts` and run `gulp readme` to update block -->
## no-invalid-regexp (ESLint: [no-invalid-regexp](http://eslint.org/docs/rules/no-invalid-regexp))

disallow invalid regular expression strings in the `RegExp` constructor (recommended)

### Usage

```json
"no-invalid-regexp": true
```
**[:straight_ruler: Rule source](https://github.com/buzinas/tslint-eslint-rules/blob/master/src/rules/noInvalidRegexpRule.ts)**
**[:blue_book: Test source](https://github.com/buzinas/tslint-eslint-rules/blob/master/src/test/rules/noInvalidRegexpRuleTests.ts)**

<!-- End:AutoDoc -->
