<!-- Start:AutoDoc:: Modify `src/readme/rules.ts` and run `gulp readme` to update block -->
## no-invalid-regexp (eslint: [no-invalid-regexp](http://eslint.org/docs/rules/no-invalid-regexp))

disallow invalid regular expression strings in the `RegExp` constructor (recommended)

### Usage

```json
"no-invalid-regexp": true
```;
<!-- End:AutoDoc -->
